src/controller/instructor/instructor.service.ts

```diff
@@ -0,0 +1,68 @@
+import { Injectable, Logger } from '@nestjs/common';
+import { db } from '../../db/index';
+import { eq, sql, inArray, and, desc, arrayContains, notInArray } from 'drizzle-orm';
+import axios from 'axios';
+import { error, log } from 'console';
+import {
```

Comment:
Please import the table scheme only if it is being utilized; otherwise, it should be removed.